### PR TITLE
#1 Image upload

### PR DESCRIPTION
**Pull request by erksch**
Implements #1

### CODE REVIEW

File: src/config/api.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'http://localhost:4822',
```
[the remark this answers is not in the document]
Comment (erksch) [reply src/config/api.js]: Done :+1:

File: src/upload/FileUpload.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  };
+
+  async submitImage(image) {
+    const formData = new FormData();
+    formData.append('image', image);
+
+    this.setState({ isUploading: true });
+
+    await api.post('/images', formData, {
```
[the remark this answers is not in the document]
Comment (erksch) [reply src/upload/FileUpload.js]: Done. I created a seperate file for image related requests.

File: src/upload/UploadDropzone.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import type { DropFilesEventHandler } from 'react-dropzone';
+import './UploadDropzone.scss';
+
+type Props = {
+  onDrop: DropFilesEventHandler,
+};
+
+function UploadDropzone(props: Props) {
```
[the remark this answers is not in the document]
Comment (erksch) [reply src/upload/UploadDropzone.js]: Done. Renamed it to `FileDropzone`.

File: src/upload/FileUpload.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      headers: {
+        'Content-Type': 'multipart/form-data',
+      },
+    })
+      .then(() => this.setState({ hasUploaded: true }))
+      .catch((error) => {
+        if (error.response) {
+          this.setState({ uploadError: error.response.message });
+        } else {
+          this.setState({ uploadError: 'Our server seems to be unavailable at the moment.' });
```
[the remark this answers is not in the document]
Comment (erksch) [reply src/upload/FileUpload.js]: You're right but we don't want to make to much upfront design by extracting the error handling somewhere. We can consider /refactor it when we add redux state or something.

File: .env.example [hunk 5]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:4822
```
Comment (Askir) [inline .env.example]: thanks <3

**Pull request merged**